File: exe10.py
class Calculadora:
    def somar_dois(self, a, b):
        return a + b

    def somar_tres(self, a, b, c):
        return a + b + c

if __name__ == "__main__":
    calc = Calculadora()
    
    resultado1 = calc.somar_dois(5, 10)
    resultado2 = calc.somar_tres(5, 10, 15)

    print("Soma de 5 e 10:", resultado1)
    print("Soma de 5, 10 e 15:", resultado2)

File: exe10.go
package main

import "fmt"

func somarDois(a int, b int) int {
    return a + b
}

func somarTres(a int, b int, c int) int {
    return a + b + c
}

func main() {
    resultado1 := somarDois(5, 10)
    resultado2 := somarTres(5, 10, 15)

    fmt.Println("Soma de 5 e 10:", resultado1)
    fmt.Println("Soma de 5, 10 e 15:", resultado2)
}
